ui/components/ui/card/README.mdx

```diff
@@ -0,0 +1,42 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
```

Comment:
What's this for? Not sure if you left it by accident or meant to wrap it in jsx?